terraform/modules/proxmox/cloudinit/bootloader.sh

```
@@ -237,11 +241,11 @@ while [ $# -gt 0 ]; do
     --telegram_chat_id)                telegram_chat_id="$2"; shift;;
     --telegram_bot_token)              telegram_bot_token="$2"; shift;;
     --infrastructure_yaml)             infrastructure_config_yaml_path="$2"; shift;;
-		(--) 		shift; break;;
-		(*) 		error "unrecognized option $1";;
-		(*)		  error "unsupport command $1";;
-	esac
-	shift
+    (--) 		shift; break;;
+    (*) 		error "unrecognized option $1";;
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 246.

(SC2221)